### ISSUE TITLE
Event start and end dates don't do what they should

### ISSUE THREAD

**Issue by jrachlin** [shevett/congo#358]
I've set event start and end dates for Arisia 2016, but the email I just received after registering myself says "Date of event:	${EventStart}" instead of January 15.

**Comment by shevett** [comment]
I believe this may be a problem with cached event data after an edit.  Need to investigate locally.

**Comment by jrachlin** [comment]
I don't think so; I seem to recall the issue happening when I created the event for A16 and input the start and end dates on creation. I may be misremembering.
